docs/apm/traces/get-started-transaction-tracing/set-up-traces-collection-for-kubernetes-environments.md

```diff
@@ -33,22 +33,21 @@ Traces will be enhanced with Kubernetes metadata, similarly to the logs and metr
 
 ## Installing Sumo Logic Tracing on Kubernetes
 
-Installation is almost the same as for the official [SumoLogic Kubernetes Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection), except a tracing flag (`sumologic.traces.enabled=true`) needs to be enabled. The process follows using a Helm chart to set all required components. It will automatically download and configure [OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which will collect, process, and export telemetry data to Sumo Logic.
+Installation is the same as for the official [SumoLogic Kubernetes Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection). The process uses a Helm chart to set all required components. It will automatically download and configure the [OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which will collect, process, and export telemetry data to Sumo Logic.
```

Comment:
```suggestion
Installation is the same as for the official [Sumo Logic Kubernetes Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection). The process uses a Helm chart to set all required components. It will automatically download and configure the [OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which will collect, process, and export telemetry data to Sumo Logic.
```